src/journal/journalctl.c

```diff
@@ -2420,10 +2428,21 @@ int main(int argc, char *argv[]) {
                 }
         }
 
+        if (arg_cursor_walk) {
+                _cleanup_free_ char *cur = NULL;
+                if (0 == read_one_line_file(arg_cursor_walk, &cur)
```

Comment:
yoda comparison we like not...

please add proper error messages here in case read_one_line_file() fails. in fact, for any function failing here...